src/app/src/components/FilterSidebarSearchTab.jsx

```diff
@@ -647,62 +723,65 @@ function FilterSidebarSearchTab({
                             onKeyPress={submitFormOnEnterKeyPress}
                         />
                     </div>
-                </div>
-                <div className="form__field">
-                    <InputLabel
-                        shrink={false}
-                        htmlFor={CONTRIBUTORS}
-                        className={classes.inputLabelStyle}
-                    >
-                        Filter by Area
-                    </InputLabel>
-                    {boundaryButton}
-                </div>
+                </ShowOnly>
                 <div className="form__action">
-                    {!embed ? (
-                        <a
-                            className="control-link inherit-font"
-                            href="mailto:info@openapparel.org?subject=Reporting an issue"
+                    {fetchingFacilities ? (
+                        <CircularProgress size={30} />
+                    ) : (
+                        <Button
+                            variant="contained"
+                            type="submit"
+                            color="primary"
+                            className={classes.font}
+                            style={{
+                                boxShadow: 'none',
+                            }}
+                            onClick={() =>
+                                searchForFacilities(vectorTileFlagIsActive)
+                            }
+                            disabled={fetchingOptions}
+                            fullWidth
                         >
-                            Report an issue
-                        </a>
-                    ) : null}
-                    <div className="offset offset-right" style={{ flex: 1 }}>
+                            Search
+                        </Button>
+                    )}
+                    {checkIfAnyFieldSelected(allFields) ? (
```

Comment:
The reset button looks odd when the search is spinning - do we want to hide it while loading data? (The picture might look a bit odd because I took it in embed mode)
<img width="326" alt="Screen Shot 2022-03-15 at 12 11 28 PM" src="https://user-images.githubusercontent.com/21046714/158423090-25001e6d-d5c3-4147-aa6a-410c83850a35.png">